worker/provisioner/provisioner.go

```diff
@@ -405,10 +405,6 @@ func (p *containerProvisioner) getRetryWatcher() (watcher.NotifyWatcher, error)
 }
 
 func (p *containerProvisioner) getProfileWatcher() (watcher.StringsWatcher, error) {
-	if p.containerType != instance.LXD {
```

Comment:
Should we add a note on why we're not checking this?  For our future selves and head scratching later?